ddb_cp.module

```diff
@@ -125,6 +125,8 @@ function ddb_cp_get_test_result($site = NULL, $number = NULL) {
   else {
     return MENU_NOT_FOUND;
   }
+
+  // @TODO: This function never returns any XML data only page not found error code?
```

Comment:
As this stage I would like the DocBlock to reflect the actual function.

The general description state that this is a menu callback which renders a raw Jenkins XML test report.

The correct return value would be int. MENU_NOT_FOUND or null could be the description.